packages/core-ethereum/src/types/channelEntry.ts

```diff
@@ -1,11 +1,16 @@
 import type { Types } from '@hoprnet/hopr-core-connector-interface'
 import BN from 'bn.js'
 import { u8aSplit, serializeToU8a, toU8a } from '@hoprnet/hopr-utils'
-import { Address } from '.' // TODO: cyclic
+import { Address, Balance } from '.' // TODO: cyclic
 import { UINT256 } from '../types/solidity'
-import { ChannelStatus } from '../types/channel'
 import { stateCounterToStatus, stateCounterToIteration, getId } from '../utils'
 
+export enum ChannelStatus {
+  CLOSED,
+  OPEN,
+  PENDING_TO_CLOSE
+}
+
 // TODO: optimize storage
 class ChannelEntry implements Types.ChannelEntry {
```

Comment:
Do we need this class at all anymore, or can we just add a serialize method to channel?